lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -221,6 +224,14 @@ type GetUTxOsStatistics = "wallets"
     :> "utxos"
     :> Get '[JSON] ApiUtxoStatistics
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/selectCoins
+type SelectCoins n = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "coin-selections"
+    :> "random"
+    :> ReqBody '[JSON] (ApiSelectCoinsData n)
+    :> PostAccepted '[JSON] (ApiCoinSelection n)
```

Comment:
**[remark]**

The initial story specifies `200 OK`. Although `201 Created` could make sense, I don't think `202 Accepted` is a good fit here. Accepted is typically good for background tasks that don't necessarily create a resource.

Reply:
> **[remark]**
> 
> The initial story specifies `200 OK`. Although `201 Created` could make sense, I don't think `202 Accepted` is a good fit here. Accepted is typically good for background tasks that don't necessarily create a resource.

Done. See https://github.com/input-output-hk/cardano-wallet/pull/1152/commits/b9479367086f7c715f1f2d020c979934cf67cea6.